spec/menu_spec.rb

```diff
@@ -0,0 +1,27 @@
+require "menu"
+
+describe Menu do
+
+  subject(:menu) { described_class.new(dishes) }
+    #injects the Hash making it easier for future updates to the menu
+  let(:dishes) do 
+    {
+      pizza: 8.99,
+      wedges: 2.50,
+      soda: 1.00
+      
+    }
```

Comment:
Great double - enough info to test while using not using all your time to create it.